src/config/mod.rs

```diff
@@ -111,11 +111,11 @@ macro_rules! set_from_flag {
 
 impl Config {
 
-    pub fn default() -> Config {
+    pub fn play_out_aftermath(play_out_aftermath: bool) -> Config {
```

Comment:
Why are you introducing this special case for `play_out_aftermath`? The fact that this is seems to be the only option that is changed during the tests isn't because it's the only one that should be tested, but because it's the only one I got around to testing. :grin: I'd prefer if we didn't have this here. Setting it in the `play_out_aftermath` function in the test modules should work easily enough, right?